src/js/utils/websockets.ts

```diff
@@ -70,7 +73,12 @@ interface PreflightEvent {
   payload: Preflight;
 }
 interface JobEvent {
-  type: 'TASK_COMPLETED' | 'JOB_COMPLETED' | 'JOB_FAILED' | 'JOB_CANCELED';
+  type:
+    | 'TASK_COMPLETED'
```

Comment:
The naming here kinda throws me off;  (understanding this comes from `push.py`) I feel like this should be named similar to the other websocket events and their subsequent actions.

```suggestion
    | 'JOB_STEP_COMPLETED'
```

Reply:
That makes sense from the front end, but I think CCI calls them "tasks" and that's throughout the back end code. Either way, not a high priority. 👍

Reply:
There are steps in the backend too. Each step of the plan runs one task.